app/javascript/dashboard/i18n/locale/en/advancedFilters.json

```
@@ -2,12 +2,17 @@
   "FILTER": {
     "TITLE": "Filter Conversations",
     "SUBTITLE": "Add filters below and hit 'Apply filters' to filter conversations.",
+    "EDIT_CUSTOM_FILTER": "Edit Custom Folder",
```

```suggestion
    "EDIT_CUSTOM_FILTER": "Edit Folder",
```

We need not say "Custom Folder" here, it's apparent